packages/remark-images-download/src/index.js

```diff
@@ -0,0 +1,43 @@
+const visit = require('unist-util-visit')
+const fs = require('fs')
+const path = require('path')
+const request = require('request')
+const shortid = require('shortid')
+const url = require('url')
+
+function plugin (opts) {
+  return (tree) => {
+    if (opts && opts.downloadImage) {
+      // images are downloaded in destinationPath
+      let destinationDir = opts.downloadDestination ? opts.downloadDestination : './'
+      if (destinationDir.slice(-1) !== '/') destinationDir += '/'
+      const destinationPath = `${destinationDir}${shortid.generate()}`
```

Comment:
I'd use the `path` module already required instead of lines 13 and 14: https://nodejs.org/api/path.html#path_path_join_paths

`const destinationPath = path.join(destinationDir, shortid.generate(), '/')` (l.13 now obsolete)